docs/data-model.rst

```diff
@@ -813,6 +820,15 @@ of this fact to compute the ``parent`` column of a mutation table, if all
 other information is valid.
 
 
+Computing mutation times
+------------------------
+
+In the case where the method generating a tree sequence does not generate mutation
+times, valid times can be provided by :meth:`TableCollection.compute_mutation_parents`.
+If all other information is valid this method will evenly distribute mutations at a
+site along their shared edges.
```

Comment:
To clarify which sort of "evenly", how about "will assign times to the mutations by placing them at evenly spaced intervals along their edge (for instance, a single mutation on an edge between a node at time 1.0 and a node at time 4.0 would be given time 2.5; while two mutations on that edge would be given times 2.0 and 3.0)."

Reply:
Yeah an example really helps. Fixed in https://github.com/tskit-dev/tskit/pull/672/commits/1514c8f4be60fad61b2bda4db12f056d5e6f7c65